config/config.yaml

```diff
@@ -0,0 +1,330 @@
+    trigger_model: 
+      model: seasonal_rainfall_forecast
+      trigger-on-minimum-probability: 0.8 # based on ensamble member probability compared against lower tercile 
+      trigger-on-minimum-probability-drought-extent: 0.6  # for drought extent using below average sesonal rain as a proxy
+      trigger-on-minimum-admin-area-in-drought-extent: 0.5 #for drought extent(grid) overlap with  admin polygons 
+    Climate_Region:
+      - name: Meher
+        climate-region-code: 1
+        leadtime:
+          Jan: 
+            - FMA: "1-month"
+          Feb: 
+            - FMA: "0-month"
+          Mar: 
+            - FMA: "0-month"
+          Apr: 
+            - FMA: "3-month"
+          May: 
+            - JAS: "2-month"
+          Jun: 
+            - JAS: "1-month"
+          Jul: 
+            - JAS: "0-month"
+          Aug: 
+            - JAS: "0-month"
+          Sep: 
+            - JAS: "0-month"
+          Oct: 
+            - FMA: "4-month"
+          Nov: 
+            - FMA: "3-month"
+          Dec: 
+            - FMA: "2-month"
+      - name: Belg
+        climate-region-code: 2
+        leadtime:
+          Jan: 
+            - FMA: "1-month"
+          Feb: 
+            - FMA: "0-month"
+          Mar: 
+            - JAS: "3-month"
+          Apr: 
+            - JAS: "2-month"
+          May: 
+            - JAS: "1-month"
+          Jun: 
+            - JAS: "0-month"
+          Jul: 
+            - JAS: "0-month"
+          Aug: 
+            - JAS: "0-month"
+          Sep: 
+            - JAS: "0-month"
+          Oct: 
+            - FMA: "4-month"
+          Nov: 
+            - FMA: "3-month"
+          Dec: 
+            - FMA: "2-month"
+      - name: Northern
+        climate-region-code: 3
+        leadtime:
+          Jan: 
+            - MAM: "2-month"
+          Feb: 
+            - MAM: "1-month"
+          Mar:
+            - MAM: "0-month"
+          Apr:
+            - JAS: "3-month"
+          May:
+            - JAS: "2-month"
+          Jun: 
+            - JAS: "1-month"
+          Jul: 
+            - JAS: "0-month"
+          Aug: 
+            - JAS: "0-month"
+          Sep: 
+            - JAS: "0-month"
+          Oct: 
+            - JAS: "5-month"
+          Nov: 
+            - JAS: "4-month"
+          Dec: 
+            - MAM: "3-month"
+      - name: Southern
+        climate-region-code: 4
+        leadtime:
+          Jan: 
+            - MAM: "2-month"
+          Feb: 
+            - MAM: "1-month"
+          Mar:
+            - MAM: "0-month"
+          Apr:
+            - MAM: "0-month"
+          May:
+            - MAM: "0-month"
+          Jun: 
+            - MAM: "0-month"
+          Jul: 
+            - OND: "3-month"
+          Aug: 
+            - OND: "2-month"
+          Sep: 
+            -OND: "1-month"
+          Oct: 
+            - OND: "0-month"
+          Nov: 
+            - OND: "0-month"
+          Dec: 
+            - MAM: "3-month"
+
+  - name: ZWE
```

Comment:
thanks for including Zimbabwe as I asked. In the meantime however I understood from @p-phung that it will not be feasible to actually include Zimbabwe pipeline in this unified pipeline already. The zimbabwe pipeline works quite differently and for example does not look ahead at the start of the wet season (Sept) but instead looks ahead at the end (April) and whether the crop will be affected by then or not. So we established we can leave it out for now and fit in hopefully an upcoming project to actually migrate it into this unified drought pipeline. Still if possible you can just leave this config in here, and simply not use it?